.github/workflows/CI.yml

```
@@ -0,0 +1,48 @@
+on: [push, pull_request]
```

```suggestion
on:
  pull_request:
    types: [opened, synchronize]
```

https://qiita.com/noriaki/items/59a4004f670fedcd6e5a

---

Fixed.